examples/Spec.hs

```diff
@@ -23,12 +23,12 @@ import Prelude.Linear
 import Test.Hspec
 import Test.QuickCheck
 
-eqList :: forall a. (Manual.Representable a, Movable a, Eq a) => List a #-> List a #-> Ur Bool
+eqList :: forall a. (Manual.Representable a, Movable a, P.Eq a) => List a #-> List a #-> Ur Bool
```

Comment:
One-letter qualifier! I know it's not your fault. But let's slate this one for being fixed, shall we?